ldap-connector-test/pom.xml

```diff
@@ -30,6 +30,10 @@
         <configuration>
           <ivyVersion>[9.3.0,]</ivyVersion>
           <engineDownloadUrl>https://developer.axonivy.com/permalink/nightly/axonivy-engine.zip</engineDownloadUrl>
+          <additionalVmOptions>
+            -Divy.Applications.ldapconnectortest.Variables.LdapConnector.Username=${activedirectory.username}
+            -Divy.Applications.ldapconnectortest.Variables.LdapConnector.Password=${activedirectory.password}
```

Comment:
@alexsuter tests are still failing on github: https://github.com/axonivy-market/ldap-connector/runs/4331195098?check_suite_focus=true

I assume that we do not override the variables

Reply:
Could be possible. II think we have the wrong key. Maybe the application name?
ldapconnectortest. let me check the build log

Reply:
hmm.. maybe the application name is "test". I would give it a try.

If this is not the case. We can shortly print the name of the application with a System.out.println(IApplication.current().getName());

Then we can see in the build log on GitHub whats the name.